### ISSUE TITLE
Not able to set association between 4512724 and 4512720

### ISSUE THREAD

**Issue by harboeknut** [easyhomesrc/homey_namronas#40]
Try to set device ID of the dimmer to one of the groups of the button switch. No response from the dimmer after the device ID is stored to the switch.

Suspect it has to do with the button switch is included as secure, and the dimmer is not.

Will upload the change of the json file to my Homey to see the effect. Will report back.

**Comment by maginawin** [comment]
Hi @harboeknut ,

Did you set up Flows for them? This is a reference video:
https://youtu.be/N78lY61okQs

**Comment by harboeknut** [comment]
Yes, but this will not allow dimming with the wall switch, only turn on and off.

Knut

**Comment by harboeknut** [comment]
Yes, but this will not allow dimming with the wall switch, only turn on and off.

Knut

**Comment by maginawin** [comment]
Hi @harboeknut ,

I found that Homey does't add `LEVEL_CHANGE` command for Zwave Light Device in the actions, I'll add it in the next few days.

**Comment by harboeknut** [comment]
Hi again!

I changed as planned to include secure to false.
I could successfully change the parameter of the wall switch under advanced and group 2 to the device ID of the dimmer.

But there occurred a new error on the dimmer afterwards... Invalid_device_by_id comes on all requests. 

This is on version 2.1.14.

Bug in this beta version??

Knut

**Comment by harboeknut** [comment]
Any news on the error?

Knut

**Comment by maginawin** [comment]
Hi @harboeknut ,

I tried to associate them in the advance settings, but they didn't work.

I'll add an 'increase/decrease brightness' action for dimmer flow so that the wall switch can change the brightness.

**Comment by harboeknut** [comment]
Exactly!

You must set the include-secure on 4512720 to false! If not it is not possible to associate the dimmer or switch with the wall switch (the latter two are included as insecure in the app.json file, since they do not support secure inclusion).

Have you had a chance to look into the Invalid_device_by_id error?

Knut

**Comment by maginawin** [comment]
Hi @harboeknut ,

OK, thanks for your advice, I'll test it.

I haven't received Invalid_device_by_id error yet.

**Comment by harboeknut** [comment]
How is the progress?

Knut

**Comment by maginawin** [comment]
Hi @harboeknut ,

I'm recoding flow cards for all devices to make the description more understandable, the new version will be released this week.

**Comment by maginawin** [comment]
Hi @harboeknut ,

I've updated the Namron App, it is now supported to change the dim of 4512724 with 4512720. I setup 4 flows:

1. Increase dim

When 4512720 Group 1 on button is held down, then 4512724 Increase dim in 10 seconds.

2. Stop increase dim 

When 4512720 Group 1 on button is released, then 4512724 Stop dim change.

3. Decrease dim

When 4512720 Group 1 off button is held down, then 4512724 Decrease dim in 5 seconds.

4. Stop decrease dim 

When 4512720 Group 1 off button is released, then 4512724 Stop dim change.

https://homey.app/a/net.filllip-namron/test/

**Comment by harboeknut** [comment]
Excellent!

But have you let the 4512720 being included as insecure as well??

Knut

**Comment by maginawin** [comment]
Hi @harboeknut ,

Yes, please install the 2.2.1 version of the Namron App and test it. It will take effect after you re-add the device.

https://homey.app/a/net.filllip-namron/test/
